themes/custom/az_barrio/theme-settings.php

```diff
@@ -24,6 +24,546 @@
  * Example on how to alter theme settings form
  */
 function az_barrio_form_system_theme_settings_alter(&$form, FormStateInterface $form_state) {
+
+  // Change collapsible fieldsets (now details) to default #open => FALSE.
+  $form['theme_settings']['#open'] = FALSE;
+  $form['logo']['#open'] = FALSE;
+  $form['favicon']['#open'] = FALSE;
+
+  // Vertical tabs.
+  $form['bootstrap'] = [
+    '#type' => 'vertical_tabs',
+    '#prefix' => '<h2><small>' . t('Bootstrap settings') . '</small></h2>',
+    '#weight' => -10,
+  ];
```

Comment:
I think this `Bootstrap settings` tab is empty / might not actually be used for anything.

```suggestion
```